distributions/otelcol-contrib/.goreleaser.yaml

```diff
@@ -0,0 +1,257 @@
+partial:
+  by: target
+project_name: opentelemetry-collector-releases
```

Comment:
Can you change this name? This should solve the problem we had during the last release.

cc @cpanato

Reply:
looks like yes because then the checksum will be with this name

```
name_template: '{{ .ProjectName }}_checksums.txt'
```

Reply:
Instead of updating the project name I went with updating the checksum directly.  `.ProjectName` is used other places and including the distribution name didnt feel right.